### PR TITLE
ADR proposal: AWS Control Tower

### PR DESCRIPTION
**Pull request by s0enke**

### CODE REVIEW

File: docs/adrs/nnnn-aws-control-tower.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ - CT implies additional costs (e.g. Config Rules)
+ - Using CWS Canaries provides an audit log of the Click Ops because of the screenshots
+ - The CT LZ setup/update/repair has to be supervised since it fails sometimes and has to be restarted (TODO)
+ - superwerker provides links to how-tos for wiring existing IdPs like GSuite, AzureAD, etc. (TODO)\
+ 
+## TODO
+
+ - [ ] How to handle SSO and Users
+ - [ ] Document CW Events / SSM Parameters
+ - [ ] Should we enable additional GuardRails? Overlaps with SH?
```
Comment (yawn) [inline docs/adrs/nnnn-aws-control-tower.md:32]: Decision: we know they exist but defer the decision (= waiting for CT support and APIs)

File: docs/adrs/nnnn-aws-control-tower.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ - [ ] Should we consider activating/deactivating VPC creation?
```
Comment (yawn) [inline docs/adrs/nnnn-aws-control-tower.md:33]: Decision: we know they are kinda useless and inconsistent (e.g. multi region setups) but we wait until we build a network feature for this.

**Review by yawn — CHANGES_REQUESTED** [review]
LGTM*

File: docs/adrs/nnnn-aws-control-tower.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ - Since CT uses AWS SSO, it is set as the default Login and AWS Account switching solution.
+
+### Account Factory 
+
+ - superwerker promotes using the CTAF for enrolling/vending new Accounts
+
+## Consequences
+
+ - superwerker only supports regions supported by CT
+ - CT implies additional costs (e.g. Config Rules)
```
[the remark this answers is not in the document]
Comment (s0enke) [reply docs/adrs/nnnn-aws-control-tower.md]: https://github.com/superwerker/superwerker/issues/60

File: docs/adrs/nnnn-aws-control-tower.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+ - Using CWS Canaries provides an audit log of the Click Ops because of the screenshots
+ - The CT LZ setup/update/repair has to be supervised since it fails sometimes and has to be restarted (TODO)
+ - superwerker provides links to how-tos for wiring existing IdPs like GSuite, AzureAD, etc. (TODO)\
+ 
+## TODO
+
+ - [ ] How to handle SSO and Users
```
[the remark this answers is not in the document]
Comment (s0enke) [reply docs/adrs/nnnn-aws-control-tower.md:32]: TIL:

> The SSOUserEmail can be a new email address, or the email address associated with an existing AWS SSO user. Whichever you choose, this user will have administrative access to the account you're provisioning.

https://docs.aws.amazon.com/controltower/latest/userguide/account-factory.html#provision-as-end-user (edited)

[the remark this answers is not in the document]
Comment (sbstjn) [reply docs/adrs/nnnn-aws-control-tower.md:32]: This topic is deferred to #21

**Pull request merged**